src/components/GraphChart/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import { LineChart, Line, XAxis, YAxis, CartesianGrid, Tooltip, Legend } from 'recharts';
+import { Stack } from '@mui/material';
+import { PLUGIN_KEYS } from '../../constants';
+import usePluginState from '../../hooks/usePluginState';
+import SensorCard from './SensorCard';
+import DeviceThermostatIcon from '@mui/icons-material/DeviceThermostat';
+import OpacityIcon from '@mui/icons-material/Opacity';
+import AirIcon from '@mui/icons-material/Air';
+import { blue, purple, red } from '@mui/material/colors';
+import useMessage from '../../hooks/useMessage';
+
+
+const GraphChart = () => {
+    const { message } = useMessage(PLUGIN_KEYS.SENSOR_RAW, '/sensor_raw', 500);
```

Comment:
Why didin't we use SensorRaw component?
SensorRaw also uses useMessage hook with the same key. There might be unusal behavior if we use in both components.